packages/core-services/src/events/Events.ts

```diff
@@ -93,6 +93,13 @@ export type EventSignatures = {
 	'notify.deleteCustomSound'(data: { soundData: ICustomSound }): void;
 	'notify.updateCustomSound'(data: { soundData: ICustomSound }): void;
 	'notify.calendar'(uid: string, data: ICalendarNotification): void;
+	'notify.messagesRead'(
+		rid: string,
+		data: {
+			until?: Date;
+			tmid?: string;
+		},
```

Comment:
this could be a plain object, just like the `messagesRead` Streamer, making both the same would be simpler IMO, like:
```
{
  rid: string;
  until?: Date;
  tmid?: string;
}
```